spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -794,6 +800,36 @@ List<Trade> fetchByActionNamedQuery(String action);
 
 This allows table names evaluated with SpEL to be used in custom queries.
 
+==== Projections
+
+Spring Data Spanner supports projections. You can define projection interfaces based
+on domain types and add query methods that return them in your repository:
+
+[source, java]
+----
+public interface TradeProjection {
+
+	String getAction();
+
+	@Value("#{target.symbol + ' ' + target.action}")
```

Comment:
what is `target`? can you explain that below?

Reply:
I believe it is not something the user or even we can decide. The SpEL for these projection functions is handled by `SpelAwareProxyProjectionFactory` and it seems that "target" name is significant. Take a look at the projections page in the spring data commons project for a better explanation. https://github.com/spring-projects/spring-data-commons/blob/master/src/main/asciidoc/repository-projections.adoc

Reply:
Haha, I understand it, I meant can you write an explanation in the doc?

Reply:
done!